source/globalCommands.py

```diff
@@ -594,28 +594,13 @@ def script_toggleReportLineNumber(self,gesture):
 		category=SCRCAT_DOCUMENTFORMATTING
 	)
 	def script_toggleReportLineIndentation(self,gesture):

```

Comment:
```suggestion
	def script_toggleReportLineIndentation(self, gesture: inputCore.InputGesture):
```

Reply:
I will apply your suggestions regarding typing in function signatures. Note however that these lines were not initially modified by this PR.

Reply:
considering the function was fully rewritten, I think it was worth doing. Thanks for accepting the changes.